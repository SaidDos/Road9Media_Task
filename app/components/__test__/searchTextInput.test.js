/* eslint-disable no-alert */
import 'react-native';
import React from 'react';
import SearchTextInput from '../textInputs/searchTextInput';

import renderer from 'react-test-renderer';

it('should renders correctly and generating a snapShot', () => {
  const onchangeText = text => {
    alert(text);
  };
  const tree = renderer
    .create(<SearchTextInput onchangeText={onchangeText} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
