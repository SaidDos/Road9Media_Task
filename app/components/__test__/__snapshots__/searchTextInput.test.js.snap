// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should renders correctly and generating a snapShot 1`] = `
<View
  style={
    Object {
      "flexDirection": "row",
      "justifyContent": "center",
    }
  }
>
  <TextInput
    allowFontScaling={true}
    onChangeText={[Function]}
    placeholder="Search by service name..."
    placeholderTextColor="black"
    rejectResponderTermination={true}
    style={
      Object {
        "alignSelf": "center",
        "backgroundColor": "#07dddd",
        "height": 50,
        "width": 710,
      }
    }
    underlineColorAndroid="transparent"
  />
  <Text
    allowFontScaling={false}
    style={
      Array [
        Object {
          "color": "black",
          "fontSize": 30,
        },
        Object {
          "alignSelf": "center",
          "position": "absolute",
          "right": 20,
        },
        Object {
          "fontFamily": "Ionicons",
          "fontStyle": "normal",
          "fontWeight": "normal",
        },
        Object {},
      ]
    }
  >
    ï’¥
  </Text>
</View>
`;
